SEACAS: link failure due to multiple definitions of io_old_termios

## Bug Report
@gsjaardema 

### Description
Recent update to seacas has broken percept link in empire builds:
```
00:24:10  packages/seacas/libraries/aprepro_lib/libaprepro_lib.a(apr_getline.cc.o):(.bss+0xc80): multiple definition of `io_old_termios'
00:24:10  packages/seacas/libraries/ioss/src/libIoss.a(Ioss_Getline.C.o):(.bss+0xc80): first defined here
00:24:10  packages/seacas/libraries/aprepro_lib/libaprepro_lib.a(apr_getline.cc.o): In function `(anonymous namespace)::search_term()':
00:24:10  cuda-opt-Volta70-static-rdc/src/Trilinos/packages/seacas/libraries/aprepro_lib/apr_getline.cc:902: multiple definition of `io_new_termios'
00:24:10  packages/seacas/libraries/ioss/src/libIoss.a(Ioss_Getline.C.o):cuda-opt-Volta70-static-rdc/src/Trilinos/packages/seacas/libraries/ioss/src/Ioss_Getline.C:237: first defined here
00:24:10  collect2: error: ld returned 1 exit status
```


### Steps to Reproduce
1. SHA1:  [insert here]
1. Configure script:  [attach here]
1. Configure log:  [attach here]
1. Build log:  [attach here]
1. Input deck:  [attach here]
1. Do this.
1. Do that.
1. Shake fist angrily at computer.
1. Run log:  [attach here]

--- a/packages/seacas/libraries/aprepro_lib/apr_getline.cc
+++ b/packages/seacas/libraries/aprepro_lib/apr_getline.cc
@@ -36,9 +36,6 @@
 #ifndef __unix__
 #define __unix__ 1
 #endif
-
-#include <termios.h>
-struct termios io_new_termios, io_old_termios;
 #endif
 
 /********************* C library headers ********************************/
@@ -121,6 +118,11 @@ namespace {
 #endif
 
 namespace {
+#ifdef __unix__
+#include <termios.h>
+  struct termios io_new_termios, io_old_termios;
+#endif
+
   void gl_char_init(void) /* turn off input echo */
   {
 #ifdef __unix__

--- a/packages/seacas/libraries/ioss/src/Ioss_Getline.C
+++ b/packages/seacas/libraries/ioss/src/Ioss_Getline.C
@@ -36,9 +36,6 @@
 #ifndef __unix__
 #define __unix__ 1
 #endif
-
-#include <termios.h>
-struct termios io_new_termios, io_old_termios;
 #endif
 
 /********************* C library headers ********************************/
@@ -121,6 +118,12 @@ namespace {
 #endif
 
 namespace {
+#ifdef __unix__
+#include <termios.h>
+  struct termios io_new_termios;
+  struct termios io_old_termios;
+#endif
+
   void gl_char_init(void) /* turn off input echo */
   {
 #ifdef __unix__

